examples/stable_diffusion_v2/text_to_image.py

```diff
@@ -156,15 +156,18 @@ def main(args):
         lora_only_ckpt=args.lora_ckpt_path,
     )
 
+    commom_kwargs = {
+        "model_type": "v" if args.version == "2.1" else "noise",
+    }
```

Comment:
According to this file, “model_type” is the only key in commom_kwargs. BTW, what’s the meaning of commom? Common?

Reply:
typo fixed; wrapping with kwargs to be extendable